### DISCUSSION TITLE
Is Django-recaptcha only usable with Google recaptcha ?

### DISCUSSION THREAD

**Discussion by kw-andy** [django-recaptcha/django-recaptcha#283] [General]
Hi,

I'm checking all the captcha solutions for django. I found the django-recaptcha solution quite interesting but I was wondering if it is only usable with google recaptcha, please?

My bet would be yes but I'm asking, just in case.

Best regards,
Andy

**Comment by Andrew-Chen-Wang** [comment]
Yes just google recaptcha

**Reply by kw-andy** [reply]
Cheers Andrew!

**Reply by tbrlpld** [reply]
I was playing with the thought of enabling H-captcha too since it should be pretty simple, but I never got around to it.
